kfdef/kfctl_openshift.v1.2.0.yaml

```diff
@@ -14,8 +14,13 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: stacks/openshift/application/istio-stack
+        path: stacks/openshift/application/istio-1-3-1-stack
```

Comment:
Do you plan to add this `istio-1-3-1-stack` to the `v1.2.0` branch?

Reply:
@LaVLaS good question, right now it will be in master.

Reply:
@LaVLaS based on your comment, I reverted kfctl for v1.2.0 back to original and added kfserving to the new master kfctl. /cc @pvaneck @animeshsingh